public/static/docs/command-reference/gc.md

```diff
@@ -33,15 +33,13 @@ restored using `dvc fetch`, as long as they have previously been uploaded with
 
 ## Options
 
-- `-a`, `--all-branches` - keep cached objects referenced from the latest commit
-  across all Git branches. It should be used if you want to keep data for the
-  latest experiment revisions. Especially, if you intend to use `dvc gc -c` this
-  option is much safer.
+- `-a`, `--all-branches` - keep cached objects referenced in all Git branches.
+  Useful for keeping data for all the latest experiment versions. This option is
+  a safer alternative to `dvc gc -c`.
```

Comment:
the last sentence has a different meaning now

Reply:
MEaaning this?
> This option is a safer alternative to `dvc gc -c`.

I didn't write or change that sentence in this PR so seems like we can leave it for the corresponding PR? I guess it's related to iterative/dvc/issues/2325

Reply:
initial meaning was - if you want to use `dvc gc -c` consider using `dvc gc -c -a`

Reply:
Git it. I changed it to ``Its recommended to consider including this option when using`-c` i.e. `dvc gc -ac`.``  Also updated `-c` option desc a little. See 6ed1d449

Reply:
*Got it